src/components/Content/Content.js

```diff
@@ -33,6 +33,16 @@ export default function Content() {
     setTodos(editedTodos);
   };
 
+  const handleChangeStatus = (id, completed) => {
+    const newStatus = todos.map((todo) => {
```

Comment:
Incorrect name for variable, `newStatus` - means single entity, but it should represent whole list of todos, check `handleEditTodo` function.